test/libsolidity/smtCheckerTests/inline_assembly/assembly.sol

```diff
@@ -0,0 +1,18 @@
+pragma experimental SMTChecker;
+
+contract C {
+	function f() internal pure returns (bool) {
+		bool b;
+		assembly { b := 1 } // This assignment is overapproximated at the moment, we don't know value of b after the assembly block
```

Comment:
We can support the assignment when the rhs is a literal or a solidity var

Reply:
As agreed, this is more complicated than it seems on the first glance.
If we continue in this direction, it will be a separate PR.